### DISCUSSION TITLE
DAI 2024-2025 - Practical work 3 - Dani Tiago Faria dos Santos and Nicolas Duprat

### DISCUSSION THREAD

**Discussion by skyrag** [heig-vd-dai-course/heig-vd-dai-course#529] [Show and tell]
le but serait de continuer notre 2 projet."Pokemon Octogone", on rajouterais donc la fonction de voir le pokedex sur le site ainsi que le modifier a l'aide de command comme dans l'exemple.

**Comment by GeraudSilvestri** [comment]
Bon pour moi
